front/app/components/admin/PostManager/components/PostPreview/Idea/AdminIdeaEdit.tsx

```diff
@@ -43,15 +42,8 @@ const AdminIdeaEdit = ({
   ideaId: string;
   goBack: () => void;
 }) => {
-  const previousPathName = useContext(PreviousPathnameContext);
-  const { data: authUser } = useAuthUser();
   const { data: idea } = useIdeaById(ideaId);
   const { mutate: deleteIdeaImage } = useDeleteIdeaImage();
-  const granted = usePermission({
-    item: idea?.data || null,
-    action: 'edit',
-    context: idea?.data || null,
-  });
```

Comment:
Not needed since we can't reach this component without having the right permissions. Permissions should not be managed at a component level in any case.